jetty-ee11/jetty-ee11-servlet/src/main/java/org/eclipse/jetty/ee11/servlet/CrossContextServletContext.java

```diff
@@ -113,19 +113,7 @@ public Set<String> getResourcePaths(String path)
     @Override
     public URL getResource(String path) throws MalformedURLException
     {
-        try
-        {
-            return _targetContext.getBaseResource().resolve(path).getURI().toURL();
-        }
-        catch (MalformedURLException e)
-        {
-            throw e;
-        }
-        catch (Throwable e)
-        {
-            // catch IOException, RuntimeException, and things like java.nio.fileInvalidPathException here.
-            throw (MalformedURLException)new MalformedURLException(path).initCause(e);
-        }
+        return _targetContext.getBaseResource().resolve(path).getURI().toURL();
```

Comment:
I don't think this should be reverted. @joakime WDYT?

Reply:
@gregw the `ee11` code isn't ensuring that an `IllegalStateException` thrown from `resolve()` isn't exposed - is this correct? The javadoc for `getResource()` only allows `MalformedURLException` to be thrown.